static/app/components/group/suspectReleases.tsx

```diff
@@ -0,0 +1,91 @@
+import styled from '@emotion/styled';
+
+import AsyncComponent from 'sentry/components/asyncComponent';
+import Placeholder from 'sentry/components/placeholder';
+import Version from 'sentry/components/version';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import type {Group, Release} from 'sentry/types';
+
+import AvatarList from '../avatar/avatarList';
+import TimeSince from '../timeSince';
+
+import SidebarSection from './sidebarSection';
+
+type Props = AsyncComponent['props'] & {
+  group: Group;
+};
+
+type State = AsyncComponent['state'] & {
+  suspectReleases: {suspectReleases: Release[]} | null;
```

Comment:
Annoying how this nests. Is this how we designed the API?

Reply:
flattened it in https://github.com/getsentry/sentry/pull/37396